python/ray/scripts/scripts.py

```diff
@@ -334,9 +335,6 @@ def start(node_ip_address, redis_address, redis_port, num_redis_shards,
         if redis_max_clients is not None:
             raise Exception("If --head is not passed in, --redis-max-clients "
                             "must not be provided.")
-        if include_webui:
-            raise Exception("If --head is not passed in, the --include-webui "
```

Comment:
Why remove this exception?

Reply:
I guess you could start a dashboard on any (or every machine), right?

Reply:
Starting the reporter process is gated behind include_webui.
This exception makes it impossible to start the reporters on the worker nodes.

Reply:
Good catch. I just pushed a small change to remove the gating. That seemed like the right approach to me (instead of passing in `--include-webui` everywhere. In principle we should be able to start the actual dashboard on any machine, so it could make sense to remove this exception, but we can do that later I suppose.